src/mmw/js/src/user/models.js

```diff
@@ -197,9 +197,52 @@ var ForgotFormModel = Backbone.Model.extend({
     }
 });
 
+var ItsiRegisterModel = Backbone.Model.extend({
+    defaults: {
+        username: null,
+        first_name: null,
+        last_name: null,
+        agreed: false,
+        clientErrors: null,
+        serverErrors: null
```

Comment:
We discussed using snake case for model attributes from now on (for parity with DRF).